libraries/interop-ribs/src/main/kotlin/com/bumble/appyx/interop/ribs/UpHandlingAppyxIntegrationPoint.kt

```diff
@@ -0,0 +1,32 @@
+package com.bumble.appyx.interop.ribs
+
+import com.badoo.ribs.util.RIBs
+import com.bumble.appyx.core.integrationpoint.IntegrationPoint
+import com.bumble.appyx.core.integrationpoint.activitystarter.ActivityStarter
+import com.bumble.appyx.core.integrationpoint.permissionrequester.PermissionRequester
+
+internal class UpHandlingAppyxIntegrationPoint(
```

Comment:
Would it make sense to add `handleUpNavigation` to the abstract `IntegrationPoint` instead?

Reply:
It is already implicitly
```kotlin
abstract class IntegrationPoint: UpNavigationHandler

fun interface UpNavigationHandler {
    fun handleUpNavigation()
}
```

Default implementation uses `Activity.finish()`, but here we need to pass invocation to RIB instead.